README.md

```diff
@@ -46,3 +46,4 @@ lua <<EOF
   })
 EOF
 ```
+To save without formatting, use `:noa w`
```

Comment:
I think it's rare that formatting isn't used formatting, so why not modify it as follows?

```suggestion
*If you save **without** formatting, use `:noa w`*
```